package.json

```diff
@@ -93,6 +92,9 @@
     "react-table": "7.7.0",
     "react-useportal": "1.0.14"
   },
+  "resolutions": {
+    "postcss": "^8.3.11"
```

Comment:
stylelint postcss-css parser needs postcss 8, but storybook has postcss v7 as dependecy, so we need to force resolution of postcss 8 for stylelint compatibility